lib/pages/home_page.dart

```diff
@@ -13,17 +15,25 @@ class _HomePageState extends State<HomePage> {
   final GlobalKey<RefreshIndicatorState> _refreshIndicatorKey =
   new GlobalKey<RefreshIndicatorState>();
   int _currentIndex = 0;
+  static FirebaseUser _firebaseUser;
 
   final PageStorageBucket bucket = PageStorageBucket();
 
   //TODO: Replace for rest of pages
   List<Widget> _pages = [
     JobsPage(key: PageStorageKey('JobsPage')),
-    LoadingAccount(),
+    ChatList(firebaseUser: _firebaseUser),
     JobsPage(key: PageStorageKey('JobsPage2')),
+    EditLocation(),
     JobsPage(key: PageStorageKey('JobsPage3')),
   ];
 
+  @override
+  void initState() {
+    super.initState();
+    getCurrentUser(FirebaseAuth.instance);
```

Comment:
`AuthProvider` here as well. This method already exists there.

Reply:
InitState cannot have async keyword, async function must be called from a separate function and let initState() execute. I instead added Provider<AuthService>.getUser() to method getCurrentUser